docs/examples/uds_auxiliary.rst

```diff
@@ -23,75 +21,52 @@ Find below a complete configuration example :
 .. code:: yaml
 
     auxiliaries:
-        uds_aux:
-            connectors:
-                com: can_channel
-            config:
-                odx_file_path: 'path/to/my/file.odx'
-                #For Vector Box, serial number and interface needs to be updated in config.ini file
-                #request and response id need to be configured in config.ini depending on the target
-                config_ini_path: 'examples/test_uds/config.ini'
-            type: pykiso.lib.auxiliaries.udsaux.uds_auxiliary:UdsAuxiliary
+      uds_aux:
+        connectors:
+            com: can_channel
+        config:
+          # you can specify your odx file by using odx_file_path parameter
+          # and instead of using send_uds_raw method use the send_uds_config
+          # for a more human readable command
+          odx_file_path: null
+          request_id : 0x123
+          response_id : 0x321
+          # uds_layer parameter is not mandatory and by default the following
+          # values will be applied:
+          # transport_protocol -> CAN
+          # p2_can_client -> 5
+          # p2_can_server -> 1
+          uds_layer:
+            transport_protocol: 'CAN'
+            p2_can_client: 5
+            p2_can_server: 1
+          # tp_layer parameter is not mandatory and by default the following
+          # values will be applied:
+          # addressing_type -> NORMAL
```

Comment:
Maybe try to explain a bit what each parameter does, or at least the different acceptable values

Reply:
It was never explain before but yes why not